docs/.vuepress/containers/conditionalBlock/conditionalBlock.js

```diff
@@ -0,0 +1,45 @@
+const { getBuildDocsFramework } = require('../../helpers');
+
+/* eslint-disable jsdoc/require-description-complete-sentence */
+/**
+ * Container used to display/hide blocks of content relevant to specific frameworks.
+ * It relies on providing the framework name(s) as arguments to the container (see 'Usage' below) and setting a
+ * `framework` field in front matter as the framework name, for example:
+ *
+ * ```
+ * ---
+ * framework: vue
+ * ---
+ * ```
+ *
+ * Usage:
+ * ```
+ * ::: only-for react
+ * Content to be displayed only for React documentation.
+ * :::
+ *
+ * ::: only-for js react vue
+ * Content to be displayed only for JS, React and Vue documentation.
+ * :::
+ * ```
+ */
+module.exports = {
+  type: 'only-for',
+  render(tokens, index) {
+    const framework = getBuildDocsFramework() || 'js';
```

Comment:
I'm using `javascript` phrase for default framework: https://github.com/handsontable/handsontable/blob/3e27d756efa7a8d774859c9827cdf0fb2acfa2c1/docs/.vuepress/helpers.js#L73-L75.

Should we sync it?

Reply:
I'll change it to `javascript` for now, but let's change it to `getDefaultFramework()` after merging both of the PRs to the epic branch.